fetchy/vaccineDataExtractor.test.ts

```diff
@@ -0,0 +1,14 @@
+import extractVaccineData from "./vaccineDataExtractor";
+import { getAllRawVaccineSites } from "../shared/vaccineDataStore";
+
+test(`Extracts vaccine data per dhb from all html files`, async () => {
```

Comment:
:nit: Template literals are unnecessary here

```suggestion
test("Extracts vaccine data per dhb from all html files", async () => {
```